src/device-registry/app.js

```diff
@@ -103,7 +103,11 @@ app.use(function(req, res, next) {
 
 app.use(function(err, req, res, next) {
   if (err.status === 404) {
-    logger.error(`this endpoint does not exist--- ${err.message}`);
+    logger.error(
+      `this endpoint does not exist --- ${err.message} --- path: ${
```

Comment:
just making the log message more descriptive. If the endpoint does not exist, then just share the path used for easy follow up by the maintenance team.